tests/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.20
 
 require (
 	github.com/stretchr/testify v1.8.4
-	github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper v1.10.15
+	github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper v1.20.10
```

Comment:
This is an invalid (DO NOT USE) version

Reply:
@shemau since you made changes to this PR, renovate will no longer update it. So can you manually update this to v1.21.0 please

Reply:
v1.21.3 is the latest with some fixes

Reply:
actually I can push this change